crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip.rs

```diff
@@ -64,7 +66,7 @@ pub(crate) mod tests;
 /// with the constant in PoolBuf which cannot be set at runtime)
 /// ^^ obviously we're no longer following the above advice..
 ///    in the case of the pool buf management, any gossips larger than
-///    16000 will now be shrunk resulting in additional memory thrashing
+///    16,000,000 will now be shrunk resulting in additional memory thrashing
```

Comment:
This change is incorrect. The value in pool_buf is still set to `16_384`. (The original comment was an approximation, off by 384). The comment is supposed to mean that we deviated from consistency. SINCE the max message size is now 16_000_000 we need to be aware that any messages over 16_384 will result in a bunch of memory allocation, and then a bunch of memory deallocation, hence the memory thrashing.